doc/how-to/support.md

```diff
@@ -3,13 +3,14 @@
 
 We recommend using the following channels for the snaps required to run MicroCloud:
 
-* For MicroCloud: `latest/[stable|candidate|edge]`
+* For MicroCloud: `2/[stable|candidate|edge]`
 * For LXD: `5.21/[stable|candidate|edge]`
-* For MicroCeph: `quincy/[stable|candidate|edge]`
-* For MicroOVN: `22.03/[stable|candidate|edge]`
+* For MicroCeph: `squid/[stable|candidate|edge]`
+* For MicroOVN: `24.03/[stable|candidate|edge]`
```

Comment:
I'm used to seeing `[]` for optional bits but in this case we mean an alternation where only one of `stable`, `candidate` or `edge` is picked. For such notation, `24.03/(stable|candidate|edge)` feels more natural... but maybe I've seen too many regexps ;)

Reply:
Agree, the `()` look more familiar. Fixed